service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -631,13 +646,78 @@ void shouldNotifyAdminOfPlacementApplicationUpload() throws Exception {
             eq(expectedParameters), eq("12345"));
     }
 
+    @Test
+    void shouldSendNotificationToPartiesWhenAddedToCaseByEmail() throws IOException, NotificationClientException {
+        final Map<String, Object> expectedParameters = getPartyAddedByEmailNotificationParameters();
+
+        List<Representative> representatives = getRepresentatives(EMAIL, PARTY_ADDED_TO_CASE_BY_EMAIL_ADDRESS);
+
+        given(partyAddedToCaseContentProvider.getPartyAddedToCaseNotificationParameters(callbackRequest().getCaseDetails(),
+            EMAIL)).willReturn(expectedParameters);
+
+        given(partyAddedToCaseContentProvider.getPartyAddedToCaseNotificationTemplate(EMAIL))
+            .willReturn(PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE);
+
+        notificationHandler.sendNotificationToPartiesAddedToCase(new PartyAddedToCaseEvent(callbackRequest(), AUTH_TOKEN, USER_ID, representatives));
+
+        verify(notificationClient, times(1)).sendEmail(
```

Comment:
very minor, but we can do `verify(notificationClient).sendEmail(...)`

Reply:
done :)